fossunited/api/proposal.py

```
@@ -265,3 +273,67 @@ def get_public_proposal_filters(
+                "creation",
+            ],
+            order_by="creation asc",
+        )
+        or []
+    )
+
+    cfp = frappe.db.get_value(EVENT_CFP, {"event": event}, ["anonymise_proposals"], as_dict=True)
+    anonymise = bool(cfp.get("anonymise_proposals")) if cfp else False
+
+    speakers_map = {}
+    if not anonymise and proposal_names:
```

Ruff (0.14.4)

Undefined name `proposal_names`

(F821)